### DISCUSSION TITLE
Problems setting max output buffer size after re-imaging with rAudio-64bit-20240408

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1662] [Fixed Issues]
After flashing a Raspberry Pi 3B+ Rev 1.3 with image rAudio-64bit-20240408, I noticed that I could not increase the max output buffer size not much past the default value (which was not high enough to allow the library to be updated correctly).  Sorry, I cannot recall the exact upper size limit I was experiencing.

I believe I had to change the audio I2S selection from HiFiBerry DAC+ Pro  to none/auto detect,  reboot, change the buffer size to the desired value, re-enable the HiFiBerry DAC+ Pro with another reboot.  My library could then be successfully updated.

I also could not enter the buffer size value directly as I had in past revisions which I miss.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by markblinkhorn** [comment]
I sidestepped this issue by manually editing  /srv/http/data/mpdconf/outputbuffer.conf
Mine now reads:

`max_output_buffer_size  "16384"`

But I'm just a rebel...
